design-documents/graph-ql/coverage/b2b/company.md

```diff
@@ -315,5 +317,6 @@ type Customer {
     team: CompanyTeam @doc(description: "Company User team data.")
     telephone: String @doc(description: "Company User phone number.")
     status: CompanyUserStatusEnum @doc(description: "Company User status.")
+    structure_id: ID! @resolver(class: "Magento\\CompanyGraphQl\\Model\\Resolver\\Company\\Customer\\CompanyStructure") @doc(description: "ID of the company structure")
```

Comment:
this is good, just remove the @resolver here. We don't need it in this repo

Reply:
Hello @cpartica
Thank you for your note 
I fixed it.
Please recheck.

Thank you in advance.